tests/dummy/app/pods/client-typed/controller.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * TODO
```

Comment:
TODO?

Reply:
These are at the top of each controller in the dummy app (not sure why, maybe added by the generate an never removed) so this was a result of copy-paste from an existing controller. I will remove it.

Reply:
Let's remove it from everywhere.

Reply:
Done.